packages/fx-core/resource/package.nls.json

```diff
@@ -133,7 +133,7 @@
   "core.migrationToArmAndMultiEnv.Message": "Teams Toolkit will upgrade your project configuration files to support the latest features. The upgrade process will not change your custom code and create the backup files in case revert is needed.\nNotice this upgrade is a must-have to continue to use current version Teams Toolkit. If you are not ready to upgrade and want to continue to use the old version, please find Teams Toolkit in Extension and install the version <= 2.10.0",
   "core.migrationV3.confirmOnly.Message": "Please confirm the upgrade",
   "core.migrationV3.Message": "Please upgrade your project to stay compatible with the latest version, your current project contains configurations from an older Teams Toolkit.\nThe auto-upgrade process will generate backups in case an error occurs.",
-  "core.migrationV3.VS.Message": "The installed version of Teams Toolkit (%s) requires configuration changes to this project that can be done automatically. Before upgrading, select Learn more to read about the required changes or select Upgrade to perform the changes automatically.",
+  "core.migrationV3.VS.Message": "The Teams Toolkit preview feature - Teams App Configuration Improvements is turned on. This project is required to be upgrade to work with the preview features. Select Learn more to read about the required changes or select Upgrade to perform the changes automatically. If you don't want to upgrade, you can turn off the preview feature.",
```

Comment:
be upgrade -> be upgraded
Learn more -> "Learn more".  quote it.
Upgrade -> "Upgrade". quote it.

Reply:
Thanks. It is fixed.